src/host/snapshot.h

```diff
@@ -58,6 +59,33 @@ namespace asynchost
       }
     }
 
+    std::optional<std::string> find_latest_snapshot()
+    {
+      std::optional<std::string> snapshot_file = std::nullopt;
+      size_t latest_idx = 0;
+
+      for (auto& f : fs::directory_iterator(snapshot_dir))
+      {
+        auto file_name = f.path().filename().string();
+        auto pos = file_name.find(fmt::format("{}.", snapshot_file_prefix));
+        if (pos == std::string::npos)
+        {
+          LOG_FAIL_FMT("File {} does not appear to be a snapshot", file_name);
```

Comment:
```suggestion
          LOG_FAIL_FMT("Ignoring \"{}\" because it does not start with {}", file_name, snapshot_file_prefix);
```

Is more explicit and actionable.